pyccel/commands/pyccel_compile.py

```diff
@@ -0,0 +1,172 @@
+    group.add_argument('--flags', type=str, \
+                       help='Additional compiler flags.')
+    group.add_argument('--wrapper-flags', type=str, \
+                       help='Additional compiler flags for the wrapper.')
+    group.add_argument('--debug', action=argparse.BooleanOptionalAction, default=None,
+                        help='Compile the code with debug flags, or not.\n' \
+                        ' Overrides the environment variable PYCCEL_DEBUG_MODE, if it exists. Otherwise default is False.')
+    group.add_argument('--include',
+                        type=str,
+                        nargs='*',
+                        dest='include',
+                        default=(),
+                        help='Additional include directories.')
+    group.add_argument('--libdir',
+                        type=str,
+                        nargs='*',
+                        dest='libdir',
+                        default=(),
+                        help='Additional library directories.')
+    group.add_argument('--libs',
+                        type=str,
+                        nargs='*',
+                        dest='libs',
+                        default=(),
+                        help='Additional libraries to link with.')
+    group.add_argument('--output', type=pathlib.Path, default = None,\
+                       help="Folder in which the output is stored (default: FILE's folder).")
+    # ...
+
+    # ... Accelerators
+    add_accelerator_selection(parser)
+    # ...
+
+    # ... Other options
+    group = parser.add_argument_group('Other options')
+    add_common_settings(group)
+    group.add_argument('--export-compiler-config', action='store_true',
+                        help='Export all compiler information to a JSON file with the given path (relative or absolute). '
+                       'This flag is deprecated and will be removed in v2.3. Please use pyccel config instead.')
+    # ...
+
+
+def pyccel(*, filename, language, output, export_compiler_config, **kwargs):
+    """
+    Call the pyccel pipeline.
+
+    Handle the deprecated --export-compiler-config command and call the pyccel pipeline.
+
+    Parameters
+    ----------
+    filename : Path
+        Name of the Python file to be translated.
+    language : str
+        The target language Pyccel is translating to.
+    output : str
+        Path to the working directory.
+    export_compiler_config : bool
+        Indicates if compiler information should be exported.
+    **kwargs : dict
+        See execute_pyccel.
+    """
+    # Imports
+    from pyccel.errors.errors     import Errors
+    from pyccel.codegen.pipeline  import execute_pyccel
+
+    errors = Errors()
+    # ...
+    # To be removed with v2.3
+    # ...
+    cext = filename.suffix
+    if export_compiler_config:
+        print("warning: The flag --export-compiler-config is deprecated and will be removed in v2.3. Please use pyccel config.", file=sys.stderr)
```

Comment:
For clarity I suggest quoting `pyccel config`:
```suggestion
        print("Warning: The flag --export-compiler-config is deprecated and will be removed in v2.3. Please use 'pyccel config' instead.", file=sys.stderr)
```